package.json

```diff
@@ -54,31 +54,73 @@
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "axios": "^0.19.0",
+    "body-parser": "^1.19.0",
+    "bunyan": "^1.8.14",
+    "chai": "^4.2.0",
```

Comment:
Isn't this for testing?

Reply:
If so... then `chai` should be in `devDependencies` and other related chai libraries too.

Reply:
nope this is just dependency creep, I will take it out